docs/content/latest/develop/ecosystem-integrations/spring.md

```diff
@@ -478,8 +478,7 @@ public class YcqlReactiveDataAccessApplication implements CommandLineRunner {
 
 ## Compatibility Matrix
 
-| **YugabyteDB Java Driver** | **Spring Data Cassandra** |
-| -------------------------- | ------------------------- |
-| 3.8.0-yb-6                 | 2.2.12.RELEASE and latest |
-| 4.6.0-yb-6                 | 3.0.6.RELEASE and latest  |
-
+| YugabyteDB Java Driver | Spring Data Cassandra |
+| :--------------------- | :-------------------- |
+| 3.8.0-yb-6 | 2.2.12.RELEASE and later |
```

Comment:
Is the whitespace change in the table rows intentional?

Reply:
Yes, whitespace changes here and elsewhere are to make the markdown easier to maintain, and don't affect the output.